conda.recipe/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+python setup.py install --single-version-externally-managed --record=record.txt
+
+# Copy non-.py files
+
+# tethys_apps
+cp -r $SRC_DIR/tethys_apps/static $SP_DIR/tethys_apps/
```

Comment:
Is there a reason you are manually copying the resource files here? If not, I think it would be better to have the setup.py do it. We have a utility function that can be used to find the resource files. Take a look at how the app setup.py do it: https://github.com/tethysplatform/tethys/blob/master/tethys_cli/scaffold_templates/app_templates/default/setup.py_tmpl

Reply:
I've moved all of the resource file handling into a `MANIFEST.in` file. The `include_package_data=True` option in the `setup.py` will look for the `MANIFEST.in` to determine which package data to include.